src/rpg_database/character_sheet/FocusesLogic.java

```diff
@@ -0,0 +1,29 @@
+package rpg_database.character_sheet;
+
+public class FocusesLogic {
+
+	private int focusImprovement;
+	private Focuses focus;
+
+	public FocusesLogic(Focuses focus) {
+		this.focus = focus;
+		this.focusImprovement = 0;
+	}
+
+	public FocusesLogic(FocusesLogic focusLogic) {
+		this.focus = focusLogic.focus;
+		this.focusImprovement = focusLogic.focusImprovement;
+	}
+
+	public int getFocusValue() {
+		return focus.getFocusValue() + focusImprovement;
+	}
+
+	public Focuses getFocus() {
+		return focus;
+	}
+
+	public void setFocusImprovement() {

```

Comment:
Egy setter mindig paramétert vár. Ezt már megbeszéltük. Ez akkor is igaz ha azt a változót átírod booleanre.

Pl. mi van ha 0-ra akarja átállítani a user?